package.json

```diff
@@ -151,5 +151,14 @@
     "deploy-set-version": "gulp deploy-set-version",
     "prettier": "prettier --write \"**/*\"",
     "prettier-check": "prettier --check \"**/*\""
+  },
+  "lint-staged": {
+    "*.{js,cjs,html}": [
+      "eslint --cache --quiet",
+      "pretty-quick"
```

Comment:
@sanjeetsuhag I notices this is taking much longer than before. I think the `pretty-quick` package is no longer needed since we added `lint-staged`. I think this line can become `prettier --write` instead.